recipes/r-tftargets/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = 'tftargets' %}
+{% set version = '1.3' %}
+
+package:
+  name: r-{{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - https://github.com/slowkow/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: e27ed65c283779b4dcda6e0704a37112a81bfedb198b3d7d036c1a80083670ca
+
+build:
+  number: 1
```

Comment:
The build number should be 0. You increment this only if a package with the same version is merged and is already available in the `bioconda` channel. Within a PR, regardless of the number of pushes, the build number should stay the same.